boxoffice/views/order.py

```diff
@@ -314,6 +315,35 @@ def format_assignee(line_item):
     return api_orders
 
 
+# TODO activate route when front-end becomes available
+# @app.route('/line_item/<line_item_id>/cancel', methods=['POST'])
+# @load_models(
+#     (Order, {'id': 'line_item_id'}, 'line_item')
+#     )
+def cancel_line_item(line_item):
+    if not line_item.is_confirmed:
+        # only confirmed line items can be cancelled
+        abort(401)
```

Comment:
Isn't 401 "authorisation required" while 403 is "unauthorized". One of them presents a login prompt.